forge-adventure/pom.xml

```diff
@@ -44,7 +44,7 @@
             <plugin>
                     <groupId>com.akathist.maven.plugins.launch4j</groupId>
                     <artifactId>launch4j-maven-plugin</artifactId>
-                    <version>1.7.25</version>
+                    <version>2.1.2</version>
```

Comment:
Why is this change in the Adventure module needed for this PR?

Reply:
Not needed indeed